lstchain/reco/dl1_to_dl2.py

```diff
@@ -597,48 +598,49 @@ def apply_models(dl1,
         disp_angle = dl2['psi']  # the source here is supposed to be in the direction given by Hillas
         disp_vector = disp.disp_vector(disp_norm, disp_angle, disp_sign)
 
-    dl2['reco_disp_dx'] = disp_vector[:, 0]
-    dl2['reco_disp_dy'] = disp_vector[:, 1]
+    if config['disp_method'] != None:
```

Comment:
I think it would be clearer to write that:
```
if config['disp_method'] is not None:
    do_common_disp_stuff
    if config['disp_method'] == 'disp_vector':
         do
    elif config['disp_method'] == 'disp_norm_sign':
        do
    else:
         raise ValueError("Unkown disp method")

```